src/components/tooltip/tooltip.js

```diff
@@ -145,11 +146,17 @@ function MdTooltipDirective($timeout, $window, $$rAF, $document, $mdUtil, $mdThe
       };
       var elementFocusedOnWindowBlur = false;
 
+      function windowScrollHandler() {
+        setVisible(false);
+      }
+      
       ngWindow.on('blur', windowBlurHandler);
       ngWindow.on('resize', debouncedOnResize);
+      document.addEventListener('scroll', windowScrollHandler, true);
```

Comment:
This is probably causing a bug on firefox (at least in 1.09) where tooltips flash and immediately get removed...